Xamarin.Forms.Platform.iOS/Renderers/ButtonRenderer.cs

```diff
@@ -151,6 +159,61 @@ void UpdateFont()
 			Control.TitleLabel.Font = Element.ToUIFont();
 		}
 
+		void UpdateMaxLines()
+		{
+			Control.TitleLabel.Lines = Element.MaxLines;
+		}
+
+		void UpdateLineBreakMode()
+		{
+#if __MOBILE__
+			switch (Element.LineBreakMode)
```

Comment:
We have this same code in the LabelRenderer. Maybe, could we refactor this part to have a common Helper and avoid duplicate code? Using the same code, if something changes when managing the LineBreakMode, it would change in the same way in both controls.